README.md

```diff
@@ -57,14 +57,6 @@ Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C
   sim_robot:=false robot_ip:=[robot ip]
   ```
 
-- The industrial calibration library builds against `libceres`, an optimization library, whose installation instructions are available [here](http://ceres-solver.org/building.html).
```

Comment:
correct, no longer needed because the [industrial calibration](https://github.com/ros-industrial/industrial_calibration) repo is no longer included.